bears/c_languages/CPPCheckBear.py

```diff
@@ -30,18 +30,26 @@ class CPPCheckBear:
 
     def create_arguments(self, config_file,
                          enable: typed_list(str) = [],
+                         language: str = '',
                          ):
         """
         :param enable:
             Choose specific issues to report. Issues that can be
             reported are: all, warning, style, performance,
             portability, information, unusedFunction,
             missingInclude
+
+        :param language:
+            Choose specific language for linting. Language can be
+            either C or C++
         """
         args = ('--template={line}:{severity}:{id}:{message}',)
         files = tuple(self.file_dict.keys())
 
         if enable:
             args += ('--enable=' + ','.join(enable),)
 
+        if language:
+            args += ('--language={language}',)
```

Comment:
CI is failing because you need to add tests for these lines.

Reply:
@jayvdb , can you guide me a bit what should language parameter would do if we provide it.
Because in tests, already defined parameters are giving error or warnings.
But I am unable to understand what would it return.

Reply:
If existings tests are failing on your PR, it is because your code is buggy.

The existing tests for CPPCheckBear work correctly in the CI on master builds and other peoples PRs.